nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/AbstractCommentManager.java

```diff
@@ -167,4 +168,20 @@ protected void setCommentPermissions(CoreSession session, DocumentModel document
         return ancestorIds;
     }
 
+    /**
+     * Notify the event of type {@code eventType} on the given {@code commentDocumentModel}.
+     *
+     * @param session the session
+     * @param commentDocumentModel the document model of the comment
+     * @param eventType the event type to publish
+     * @implSpec This method uses internally {@link #notifyEvent(CoreSession, String, DocumentModel, DocumentModel)}
+     * @since 11.1
+     */
+    protected void notifyEvent(CoreSession session, DocumentModel commentDocumentModel, String eventType) {
```

Comment:
Is it intentional to switch the parameter order comared to `#notifyEvent(CoreSession, String, DocumentModel, DocumentModel)`?

Reply:
yes i made it intentionally because from my  point view i started from the big object and so on. `Session` -> `Document` -> `Event`

Reply:
OK, but it's kind of confusing comparing both APIs.

Reply:
yes i see, i should i asked my self the same question we can discuss with the team to find the best approche in these uses cases where sometimes i want to change the signature of a new method comparing to an old one like this case where from my point of view is more logical to have the parameters by orders of there responsibility :)